pkg/templates/git.go

```diff
@@ -138,14 +142,21 @@ func syncTemplateFromRef(
 		return err
 	}
 
-	schema, err := loadTerraformTemplateSchema(w.Filesystem.Root())
+	// Try to get version from reference.
+	ref := repo.Reference
+	if v, err := version.NewVersion(repo.Reference); err == nil {
+		ref = v.Original()
```

Comment:
What's this operation used for? `v.Original()` seems to be the same as the reference passed in.

Reply:
Use the original ref to prevent duplicated, this is the original PR add this change the first time. https://github.com/seal-io/walrus/pull/1135